cmd/tetragon/main.go

```diff
@@ -808,6 +809,13 @@ func execute() error {
 			if err := option.ReadAndSetFlags(); err != nil {
 				logger.Fatal(log, "Failed to parse command line flags", logfields.Error, err)
 			}
+			// Override perf ring buffer choice if only the perf ring is available.
+			// NB: can't do this in option.ReadAndSetFlags() as it causes an import cycle.
```

Comment:
mmh this is annoying (indeed not a huge deal), we should be able to call the `kernels` pkg from this if we have a function to validate flags and stop tetragon starting. It can be useful in the future, maybe that's worth investigating and fixing?